turbinia/client.py

```diff
@@ -357,10 +357,10 @@ def get_task_data(
     if user:
       func_args.update({'user': user})
 
-    response = None
+    response = {}
     retry_count = 0
     credential_error_count = 0
-    while response is None and retry_count < MAX_RETRIES:
+    while not response and retry_count < MAX_RETRIES:
```

Comment:
Small nitpick, but should this be `<=` instead of `<` to include the max number

Reply:
I think as is it reflects the intent of the `MAX_RETRIES` value -- Since the count starts at 0, it will loop a total of ten times for 0-9.  I could change it to `<= MAX_RETRIES -1`, but I think that's pretty equivalent.

Reply:
Oh yeah you are right, nvm that makes total sense